src/frequenz/channels/_select.py

```diff
@@ -378,7 +376,7 @@ async def select(  # noqa: DOC503
         import datetime
         from typing import assert_never
 
-        from frequenz.channels import ReceiverStoppedError, select, selected_from
```

Comment:
This shouldn't be removed, or the example needs to be updated to use `Receiver.triggered()` instead.

Reply:
oops, replaced with `Receiver.triggered()`